fr/administrator-guide/configuring-the-catalog/inspire-configuration.rst

```diff
@@ -1,4 +1,36 @@
 .. _inspire-configuration:
 
 Configurer pour la directive INSPIRE
-####################################
\ No newline at end of file
+####################################
+
+La configuration d'un service CSW conforme INSPIRE se réalise au niveau de la 
+section "Administration > Paramètres > CSW" comme vu précédemment (voir 
+`Configurer le CSW <./csw-configuration.html>`_). Sur base du formaulaire de 
```

Comment:
Better to use :ref: See http://docs.titellus.net/geonetwork/en/contributing/writing-documentation.html#reference-to-a-section-within-a-file